tiffslide/tiffslide.py

```diff
@@ -745,14 +745,13 @@ def recover_mpp(cls, series: TiffPageSeries) -> dict[str, Any]:
             md["tiff.XResolution"] = float(x_resolution)
             md["tiff.YResolution"] = float(y_resolution)
 
-            RESUNIT = tifffile.TIFF.RESUNIT
             scale = {
-                RESUNIT.INCH: 25400.0,
-                RESUNIT.CENTIMETER: 10000.0,
-                RESUNIT.MILLIMETER: 1000.0,
-                RESUNIT.MICROMETER: 1.0,
-                RESUNIT.NONE: None,
-            }.get(resolution_unit, None)
+                "INCH": 25400.0,
+                "CENTIMETER": 10000.0,
+                "MILLIMETER": 1000.0,
+                "MICROMETER": 1.0,
+                "NONE": None,
+            }.get(resolution_unit.name, None)
```

Comment:
Hi @stergioc 

I think `tifffile.TIFF.RESUNIT` is deprecated in favour of `tifffile.RESUNIT`.

Could you adjust your PR to avoid the string keys, and use the enums?

Thanks ☺️